<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QCVECheckApp</class>
 <widget class="QMainWindow" name="QCVECheckApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QCVECheckReport</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdiArea">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="activationOrder">
       <enum>QMdiArea::CreationOrder</enum>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="separator"/>
    <addaction name="actionImport_CVE_DB"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout_QCVECheckReport"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QDockWidget" name="dockWidgetMenu">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>1</height>
    </size>
   </property>
   <property name="font">
    <font>
     <bold>false</bold>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetMenuContents">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item alignment="Qt::Qt::AlignTop">
      <widget class="QToolBox" name="dockWidgetVerticalToolBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>450</height>
        </size>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="statusTip">
        <string/>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
       <property name="accessibleName">
        <string/>
       </property>
       <property name="accessibleDescription">
        <string/>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <widget class="QWidget" name="dockWidgetVerticalToolBoxPage1">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>198</width>
          <height>419</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Menu</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item alignment="Qt::Qt::AlignTop">
          <widget class="QComboBox" name="comboBoxReports">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>180</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
         <item alignment="Qt::Qt::AlignTop">
          <widget class="QGroupBox" name="groupBoxMain">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>1</horstretch>
             <verstretch>1</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>180</width>
             <height>100</height>
            </size>
           </property>
           <property name="title">
            <string>Main</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignHCenter|Qt::AlignTop</set>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="pushButtonOpen">
              <property name="text">
               <string>Open</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonExportReport">
              <property name="text">
               <string>Export Report</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBoxCVEReport">
           <property name="minimumSize">
            <size>
             <width>180</width>
             <height>180</height>
            </size>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="title">
            <string>CVE Report</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignHCenter|Qt::AlignTop</set>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QPushButton" name="pushButtonGeneral">
              <property name="text">
               <string>General</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonSummary">
              <property name="text">
               <string>Summary</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonPackages">
              <property name="text">
               <string>Packages</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonCVEs">
              <property name="text">
               <string>CVEs</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonIgnoredCVEs">
              <property name="text">
               <string>Ignored CVEs</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item alignment="Qt::Qt::AlignTop">
          <widget class="QGroupBox" name="groupBoxCVEData">
           <property name="minimumSize">
            <size>
             <width>180</width>
             <height>70</height>
            </size>
           </property>
           <property name="title">
            <string>CVE Data</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignHCenter|Qt::AlignTop</set>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QPushButton" name="pushButtonCVEData">
              <property name="text">
               <string>Search CVEs</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionImport_CVE_DB">
   <property name="text">
    <string>Import CVE DB...</string>
   </property>
  </action>
  <action name="actionAbout_QCVECheckReport">
   <property name="text">
    <string>&amp;About QCVECheckReport</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
